### PR TITLE
Added ViT tutorial

### PR DESCRIPTION
**Pull request by vfdev-5**
cc @melissawm

### CODE REVIEW

File: docs/JAX_Vision_transformer.md [hunk 1]
```diff
@@ -0,0 +1,557 @@
...
+print("Predictions shape: ", y.shape)
+```
+
+In the following sections we set up a image classification dataset and train this model.
+
++++
+
+## Food 101 dataset
+
+In the this tutorial we use [Food 101](https://huggingface.co/datasets/ethz/food101) dataset which consists of 101 food categories, with 101'000 images. For each class, 250 manually reviewed test images are provided as well as 750 training images. On purpose, the training images were not cleaned, and thus still contain some amount of noise. This comes mostly in the form of intense colors and sometimes wrong labels. All images were rescaled to have a maximum side length of 512 pixels.
```
Comment (jakevdp) [inline docs/JAX_Vision_transformer.md:177]: `101,000`

**Review by jakevdp — COMMENTED** [review]
Looks great overall! Since the final accuracy is so low, I wonder if it would be worth augmenting this tutorial with some fine-tuning to improve the model? That's something we've alluded to so far in the examples, but haven't yet really dug into

**Comment by vfdev-5** [comment]
> I wonder if it would be worth augmenting this tutorial with some fine-tuning to improve the model? 

In this case we should fetch trained weights from somewhere, map and load them.

Maybe, simply I can train for longer and get better results...

**Comment by vfdev-5** [comment]
@jakevdp I tried various configs, now reducing the model size, image size to 120px and still training for 50 epochs I could get 67% accuracy. Maybe, it is OK for now.

**Review by jakevdp — APPROVED** [review]
Thanks!
